kustomize/internal/commands/edit/add/addtransformer.go

```diff
@@ -0,0 +1,78 @@
+// Copyright 2019 The Kubernetes Authors.
+// SPDX-License-Identifier: Apache-2.0
+
+package add
+
+import (
+	"errors"
+	"log"
+
+	"github.com/spf13/cobra"
+	"sigs.k8s.io/kustomize/api/filesys"
+	"sigs.k8s.io/kustomize/kustomize/v3/internal/commands/kustfile"
+	"sigs.k8s.io/kustomize/kustomize/v3/internal/commands/util"
+)
+
+type addTransformerOptions struct {
+	transformerFilePaths []string
+}
+
+// newCmdAddTransformer adds the name of a file containing a transformer to the kustomization file.
+func newCmdAddTransformer(fSys filesys.FileSystem) *cobra.Command {
+	var o addTransformerOptions
+
+	cmd := &cobra.Command{
+		Use:   "transformer",
+		Short: "Add the name of a file containing a transformer to the kustomization file.",
```

Comment:
s/transformer/transformer configuration/

Reply:
Fixed this and the example in all.go